package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "root",
+  "private": true,
+  "devDependencies": {
+    "husky": "^4.2.5",
+    "lerna": "^3.22.1"
+  },
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "start": "lerna run start"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint --since master",
```

Comment:
instead of writing yarn lint here, can't we add a "pre-commit" script inside each folder and perform actions depending on the folder. We can use yarn pre-commit in the outer folder

Reply:
Yes we can do this. Have added pre-commit and pre-push script inside each folder